app/femr/ui/views/triage/index.scala.html

```diff
@@ -45,10 +45,17 @@ <h2 class="text-center">Check In - Triage</h2>
                 @inputText("First Name", "firstName", true, if(viewModel != null) viewModel.getPatient.getFirstName else null, "text")
                 @inputText("Last Name", "lastName", true, if(viewModel != null) viewModel.getPatient.getLastName else null, "text")
                 @inputText("Address", "address", false, if(viewModel != null) viewModel.getPatient.getAddress else null, "text")
+
+
+                @inputText("Phone No.", "phoneNo", true, if(viewModel != null) viewModel.getPatient.getPhoneNo else null, "text")
```

Comment:
@Ayush666 change the third parameter to false because this field shouldn't be required (this should remove the red star in the user interface)